mappings/net/minecraft/util/math/MutableBox.mapping

```diff
@@ -28,23 +28,23 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
 		ARG 1 other
 	METHOD method_14658 toNbt ()Lnet/minecraft/class_2495;
 	METHOD method_14659 getSize ()Lnet/minecraft/class_2382;
-	METHOD method_14660 getBlockCountX ()I
-	METHOD method_14661 translate (III)V
+	METHOD method_14660 getXSize ()I
+	METHOD method_14661 offset (III)V
 		ARG 1 dx
 		ARG 2 dy
 		ARG 3 dz
 	METHOD method_14662 contains (Lnet/minecraft/class_2382;)Z
 		ARG 1 vec
-	METHOD method_14663 getBlockCountY ()I
-	METHOD method_14664 getBlockCountZ ()I
+	METHOD method_14663 getYSize ()I
+	METHOD method_14664 getZSize ()I
 	METHOD method_14665 empty ()Lnet/minecraft/class_3341;
 	METHOD method_14666 create (IIIIII)Lnet/minecraft/class_3341;
-		ARG 0 x1
-		ARG 1 y1
-		ARG 2 z1
-		ARG 3 x2
-		ARG 4 y2
-		ARG 5 z2
+		ARG 0 minX
+		ARG 1 minY
+		ARG 2 minZ
+		ARG 3 maxX
+		ARG 4 maxY
+		ARG 5 maxZ
 	METHOD method_14667 createRotated (IIIIIIIIILnet/minecraft/class_2350;)Lnet/minecraft/class_3341;
```

Comment:
We could consider renaming this to `rotated` maybe ?

Reply:
This returns a new instance with the rotation, rather than this one. I don't feel simply calling it `rotated` would adequately portray that.

Reply:
`rotate` would be the mutating method IMO

Reply:
Again, this method does not mutate the box. Returns a new box.

Reply:
Which `rotated` adequately portrays in my opinion.

Reply:
`rotate` != `rotated`
Rotate implies mutation to a rotated state, rotated implies the creation of a rotated copy